www/settings/custom_metrics/imgs-too-large.js

```diff
@@ -0,0 +1,34 @@
+var imgsTooLarge = [];
+var allImgs = document.querySelectorAll("img");
```

Comment:
Might be cool to inspect background images too, in a follow up diff maybe

Reply:
yeah, with bg size I guess? harder to parse but I'm interested